Code/GraphMol/Descriptors/test.cpp

```diff
@@ -1929,6 +1943,31 @@ void testPropertyQueries() {
   }
 }
 
+void testStereoCounting() {
+  const bool debugParse=false;
+  const bool sanitize=false;
+  ROMol *m = SmilesToMol("NC(C)(F)C(=O)O", debugParse, sanitize);
+  TEST_ASSERT(!m->hasProp(common_properties::_StereochemDone));
+  
+  std::vector<std::string> names;
+  names.push_back("NumAtomStereoCenters");
+  names.push_back("NumUnspecifiedAtomStereoCenters");
+  Properties prop(names);
+
+  try {
+    prop.computeProperties(*m);
+    TEST_ASSERT(0); // didn't catch exception
+  } catch (ValueErrorException) {
+    BOOST_LOG(rdErrorLog) << "---Caught stereo value error---" << std::endl;
+  }
+
+  delete m;
+  m = SmilesToMol("NC(C)(F)C(=O)O", sanitize);
```

Comment:
The result here is correct, but you have bitten yourself with the local variables (a downside to using named variables instead of just passing them in). You're passing the `sanitize` variable to the `debugParse` argument.
You want defaults here, just use: `m = SmilesToMol("NC(C)(F)C(=O)O"`